docs/contributing/templates/c2c-source.md

```diff
@@ -1,110 +1,139 @@
 ---
 id: c2c-source  #example: druva-source
 title: XYZ Source #example: Druva Source
+image: 'https://app_icons.s3.amazonaws.com/dropbox.svg' #app logo
 sidebar_label: XYZ #example: Druva
-description: Description goes here. #example: Learn how to configure the Druva Cloud-to-Cloud Source in your Sumo Logic environment.
+slug: /c2c/c2c-name #/c2c/druva
+tags:
+  - cloud-to-cloud
+  - c2c-source  #example: Druva
+description:  Description goes here. #example: Learn how to configure the Druva Cloud-to-Cloud Source in your Sumo Logic environment.
 ---
 
+import CodeBlock from '@theme/CodeBlock';
+import ExampleJSON from '/img/c2c/dropbox/example.json';
+import MyComponentSource from '!!raw-loader!/img/c2c/dropbox/example.json';
+import TerraformExample from '!!raw-loader!/img/c2c/dropbox/example.tf';
```

Comment:
@JV0812, these lines are causing the following error when I attempt to build locally:
> ERROR in ./docs/contributing/templates/c2c-source.md 102:0-75
Module not found: Error: Can't resolve 'raw-loader' in '/Users/jpipkin/Documents/GitHub/sumologic-documentation/docs/contributing/templates'

Reply:
Thanks @jpipkin1. I have fixed this now.